### PR TITLE
Book styling

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
Add styling to the application to make it match this wireframe. Built with bootstrap.

### CODE REVIEW

**Comment by Ugboaja-Uchechi** [comment]
All necessary linters have passed

File: main.js [hunk 1]
```diff
@@ -23,12 +23,18 @@ const selectors = {
 
 class UI {
```
Comment (rloterh) [inline main.js:24]: - [x] The styling of your table does not conform to the database table **alternate row colors**, as can be seen in the original wireframe below. Kindly ensure to improve upon this aspect.

Your design:
image

**The wireframe design:**
image

File: main.js [hunk 2]
```diff
@@ -23,12 +23,18 @@ const selectors = {
...
-    <td>${book.isbn}</td>
-    <td><button class="delete">Remove</button></td>
+     <div class="d-flex align-items-center justify-content-between w-100">
+     <div class="d-flex w-75 align-items-center text-center">
+    <h5 class="mx-3 w-25">"${book.title}"</h5>
+    <h5 class="mx-3 w-25"> by </h5>
+    <h5 class="mx-3 w-25">${book.author}</h5>
+    <h5 class = "ml-3 w-25"> ISBN: ${book.isbn}</h5>
+    </div>
+  <div> <button class="delete">Remove</button></div>
```
Comment (rloterh) [inline main.js]: - [ ] OPTIONAL
Entries still show up when removed and then the page is refreshed. Kindly endeavor to improve upon this aspect

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Victor & Stephanie, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ugboaja-Uchechi** [comment]
Fixed required code reviews.
All linters have been checked.

**Review by carlos-ssh — APPROVED** [review]
Hi @Ugboaja-Uchechi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**